### ISSUE TITLE
Website selector control should be usable with keyboard

### ISSUE THREAD

**Issue by mattab** [matomo-org/matomo#9601]
Goal of this issue is to make the Website selector control in Piwik App usable by keyboard.
### Reproduce
- In Piwik press TAB key
- Tab into the Website selector
- Press ENTER key -> the website selector expands and shows list of website names
- Press TAB key...
  - Got: the next control (Calendar) is selected
  - Expected: on pressing TAB, it should tab through the websites names below the selector. On pressing ENTER then it should select this website and reload the page to select this website.

**Comment by mattab** [comment]
it's been working fine for a few years already.
